docs/sphinx/about/whats-new.txt

```diff
@@ -1,13 +1,14 @@
 Version history
 ===============
 
-5.2.0-m4 (2016 June 24)
+5.2.0-m5 (2016 July 7)
 -----------------------
 
 Top-level Bio-Formats API changes:
 
 * Java 1.7 is now the minimum supported version
 * the native-lib-loader dependency has been bumped to version 2.1.4
+* the xalan dependency has been bumped to version 2.7.2
```

Comment:
@ctrueden: `xalan/serializer 2.7.2` and `native-lib-loader 2.1.4` should be the only non-OME dependency version bumps in Bio-Formats 5.2.0 due in a few days. Is there a specific process on the Fiji deployment side or should we coordinate before pushing the new Bio-Formats JARs?